x/staking/types/params.go

```diff
@@ -35,7 +35,7 @@ const (
 var (
 	KeyUnbondingTime     = []byte("UnbondingTime")
 	KeyMaxValidators     = []byte("MaxValidators")
-	KeyMaxEntries        = []byte("KeyMaxEntries")
+	KeyMaxEntries        = []byte("MaxEntries")
```

Comment:
This is a breaking change, do I need to add a changgelog entry?

Reply:
No need to add this to the changelog.